packages/acceptance-tests/pkg-tests-specs/sources/pnp.test.js

```diff
@@ -1693,4 +1693,43 @@ describe(`Plug'n'Play`, () => {
       await expect(source(`require.resolve('pkg')`)).resolves.toEqual(npath.fromPortablePath(`${path}/package/index.js`));
     }),
   );
+
+  test(
+    `it should not loose the pnpapi on portals with virtual paths`,
+    makeTemporaryEnv({}, async ({path, run, source}) => {
+      const portalTarget = await createTemporaryFolder();
```

Comment:
Better use the `xfs.mktempPromise` callback-style, as it'll remove the trash before exiting